document-models/atlas-exploratory/src/reducers/context.ts

```diff
@@ -8,17 +8,23 @@ import { type AtlasExploratoryContextOperations } from "../../gen/context/operat
 
 export const reducer: AtlasExploratoryContextOperations = {
   addContextDataOperation(state, action, dispatch) {
-    // TODO: Implement "addContextDataOperation" reducer
-    throw new Error('Reducer "addContextDataOperation" not yet implemented');
+  
+    state.originalContextData = state.originalContextData.filter(
+      (ocd) => ocd.id !== action.input.id,
+    );
```

Comment:
this is for add context data, here you're removing data

Reply:
you are right, done